README.md

```diff
@@ -1,6 +1,7 @@
 # Cryostat3
 
 [![CI build and push](https://github.com/cryostatio/cryostat3/actions/workflows/push-ci.yaml/badge.svg)](https://github.com/cryostatio/cryostat3/actions/workflows/push-ci.yaml)
+[![Quay Repository](https://quay.io/repository/cryostat/cryostat/status "Quay Repository")](https://quay.io/repository/cryostat/cryostat)
```

Comment:
I think the `container:none` badge isn't fitting? Maybe we could do something like `https://img.shields.io/badge/Quay_Repository-cryostat/cryostat-blue.svg` instead?

Reply:
It's an option. The `none` is because Quay's badges use that status only to indicate when the container is built from a Dockerfile/Containerfile directly by Quay infrastructure. It's unfortunate that it doesn't update for repositories that builds are pushed to from outside (ex. our GH Actions CI).

Reply:
Updated to use this badge, changed the colour so it is distinct from the mailing list badge, and added hover/alt-text to the badges.